components/date-time/style.scss

```diff
@@ -122,3 +121,40 @@ $datepicker__border-color: $light-gray-500;
 .react-datepicker__close-icon::after {
 	background-color: theme( primary );
 }
+
+.date-time-form {
+	display: flex;
+	justify-content: center;
+	align-items: right;
+	margin-top: 10px;
+
+	.input-container {
+		padding: 0 4px;
+		flex: 0 1 auto;
+		order: 1;
+		&.day {
+			order: 0;
+		}
+
+		input[name=day] {
+			width: 40px;
+		}
+
+		input[name=year] {
+			width: 55px;
+		}
+
+		input[name=month] {
+			width: 90px;
+		}
+	}
+}
+
+.datepicker-container {
+	margin-top: 20px;
+}
+
+/* Makes the month appear before the day for those in the USA */
+html[lang="en-US"] .date-time-form .input-container.month {
+	order: 0;
```

Comment:
we can't use the flex order property, as for a11y visual order should always match DOM order when it affects meaning and operability

Reply:
In this case, the date validation (which is done on-the-fly) is based on the month, so this is the best thing I could come up with.

This means the user will always be able to tab into the month, update it and then enter the date.